<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGSBOSrcTabDialog</class>
 <widget class="QDialog" name="CGSBOSrcTabDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="m_NullRadioButton">
     <property name="text">
      <string>x:无效标</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_OneRadioButton">
     <property name="text">
      <string>1:跳闸矩阵表</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_TwoRadioButton">
     <property name="text">
      <string>2:开关量表</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_ThreeRadioButton">
     <property name="text">
      <string>3：模拟量表</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CGSBOSrcTabDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CGSBOSrcTabDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
